### ISSUE TITLE
[Zoom] attribute is not working

### ISSUE THREAD

**Issue by niightly** [stephanrauh/ngx-extended-pdf-viewer#84]
I'm trying to use the property `[zoom]="'page-width"` but for some reason, it always renders the viewer with `automatic` type of zoom.

did I miss something from the documentation?

```html
<ngx-extended-pdf-viewer (pagesLoaded)="fetchTotalPages($event)" (pageRendered)="changePage($event)" [zoom]="'page-width'" [page]="1" [src]="'assets/Course.pdf'" [backgroundColor]="'#474747'" [showScrollingButton]="false" useBrowserLocale="true" height="{{ pdfReaderHeight }}" delayFirstView="1000" [showOpenFileButton]="false" [showBookmarkButton]="false" [showSidebarButton]="false" [showSecondaryToolbarButton]="false"></ngx-extended-pdf-viewer>
```

**Comment by niightly** [comment]
Does anyone know about it?

**Comment by stephanrauh** [comment]
I'm afraid I haven't understood the question yet. Hint: The title of the issue mentions `[zoom]`, but the bug description doesn't. Maybe you've simple forgotten a word somewhere.

**Comment by niightly** [comment]
> I'm afraid I haven't understood the question yet. Hint: The title of the issue mentions `[zoom]`, but the bug description doesn't. Maybe you've simply forgotten a word somewhere.

Oh my mistake, I was thinking about one problem and wrote about another, just edit the question... 

Can you help me pls?

**Comment by stephanrauh** [comment]
Weird bug. Seems like pdf.js has changed the way they manage the default settings. I've solved the bug with 0.9.40.
